### DISCUSSION TITLE
Migrating from semantic to fomantic

### DISCUSSION THREAD

**Discussion by etshy** [fomantic/Fomantic-UI#126] [General] [answered]
Maybe the questions is dumb, but can I migrate simply from semantic to this repo ?
Like all the classes/rules are the same ?

If this fork can do its own updates + merge official few releases there are now, it could be awesome !

**Comment by y0hami** [comment]
Yes, you should be able to replace your `semantic-ui` dependency with `fomantic-ui`. To use new components such as `calendar`, `range` etc. you will need to update your `theme.config`. The code base for FUI is exactly the same as SUI.

As for the new SUI release we have merged every new release from SUI so far, the latest SUI 2.4 release will also be merged with our 2.6 update which will (🤞) happen tomorrow.

**Comment by etshy** [comment]
Oh nice, I'll wait a bit to change then.

About the installation, with `semantic.json` and the custom semantic sources destination, is it still the same file (`semantic.json`) ?

Question is about : Do i need to install from scratch, or juste changing the dependency in the `package.json` and `update` is sufficient ?

**Comment by y0hami** [comment, marked as answer]
The `semantic.json` file is still the same, everything is the same in the code. The only differences are bug fixes and new features which we have fixed/added. We have only renamed stuff to Fomantic if it isn't related and won't break code/usage.

No, you can change the dependency and run `npm update` then rebuild. As I said if you want to use new components you will need to add them to your `theme.config`.

**Comment by etshy** [comment]
Ok, perfect Thanks a lot for answers.
